designdocs/privacy-design.md

```diff
@@ -0,0 +1,163 @@
+   </td>
+  </tr>
+  <tr>
+   <td>On a device with configurable interpupillary distance, XRSession may create an <a href="https://immersive-web.github.io/webxr/#xrviewerpose-interface">XRViewerPose</a> that will return an <a href="https://immersive-web.github.io/webxr/#xrview">XRView</a> for each eye
+   </td>
+   <td>Site access to IPD data may allow user <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-profiling">profiling</a> and <a href="https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#user-fingerprinting">fingerprinting</a>.
+   </td>
+  </tr>
+</table>
+
+#### Camera Data and XRSession Data
+The combination of camera data (e.g. using [getUserMedia](https://www.w3.org/TR/mediacapture-streams/#dom-mediadevices-getusermedia())) with [XRViewerPose](https://immersive-web.github.io/webxr/#xrviewerpose) data that is based on real-world viewer position and orientation may expose threat vectors related to [real-world geometry](https://github.com/immersive-web/privacy-and-security/blob/master/EXPLAINER.md#real-world-geometry) that are not present when only camera data is available.
+
+Such threat vectors assume that both types of data are available within a sufficiently short time interval that, given a camera frame, the viewer pose can be known at the time the frame was captured.
+
+It is suggested that the user agent either prevent access to both types of data on the same [origin](https://www.w3.org/TR/2009/WD-html5-20090423/browsers.html#origin-0) within a short time interval (e.g. 2 seconds), or inform the user of the threat vectors and obtain user consent before making both types of data available.
+
+### Private Browsing Modes
+User agents may support a mode (e.g., private browsing) of operation intended to preserve user anonymity and/or ensure records of browsing activity are not persisted on the client.
+
+There are <span style="text-decoration:underline;">no</span> additional requirements for such modes, as there is no persistent data or unique user identifier data generated by the WebXR Device API.
+
+## Background
+### Threats and Mitigations
+The following threats, and associated mitigations, are the basis of the above privacy requirements.
+
+#### Pose Data
+[XRViewerPose](https://immersive-web.github.io/webxr/#xrviewerpose) might be used for [input sniffing](https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#reading-inputs-or-input-sniffing). [Same Origin](https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#same-origin-or-single-origin-only) and [Focused and Visible](https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#focused-and-visible) mitigations address this threat.
```

Comment:
Shouldn't this be inclusive of all XRPose data? (same comment for all the bullets below)

Reply:
Good observation, this could use some clarification (the above recommendations do indeed cover both XRPose and XRViewerPose).

Reply:
Updated where applicable, I'm going to resolve this one. Thanks Nell!